index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Alice Cieslik . Persönliche Website</title>
+  </head>
+  <body>
+    <header>
+      <p><strong>Alice Cieslik . Persönliche Website</strong> </p>
+
+      <p>Ich baue eine <strong>Website</strong>.</p>
+    </header>
+
+    <nav>
+      <strong>Navigation</strong> 
+      <ul>
+        <li>Home</li>
+        <li>Über mich</li>
+        <li>Dienstleistungen</li>
+        <li>Blog</li>
+        <li>Kontakt</li>
+      </ul>
+    </nav>
+
+    <main>
+      <h1>Hier steht der Hauptinhalt</h1>
+      <h2>Home</h2>
+
+      <p> Lorem ipsum, <br> dolor sit amet consectetur adipisicing elit. Aliquam similique nam expedita fugit porro doloremque minus? Consequuntur magni laborum magnam accusantium voluptatem! Vitae non possimus iure cupiditate quo maxime inventore?
+      </p>
+
+      <h2>Über mich</h2>
+
+      <p> Lorem ipsum, <br> dolor sit amet consectetur adipisicing elit. Aliquam similique nam expedita fugit porro doloremque minus? Consequuntur magni laborum magnam accusantium voluptatem! Vitae non possimus iure cupiditate quo maxime inventore?
+    </p>
+
+    <h2>Dienstleistungen</h2>
+    <p>
+      <ol>
+        <li>Eins</li>
+        <li>Zwei</li>
+        <li>Drei</li>
+        <li>sortierte Liste</li>
+      </ol></p>
+
+    <h2>Blog</h2>
+
+    <p>Kurzvorschau aktuelle Artikel</p>
+
+    <h2>Kontakt</h2>
+
+  
+    </main>
+
+    <footer>
+      <p>Copyright und Link zum Impressum</p>
+    </footer>
+
+    <address>Kontaktdaten
+  <p>
+      <ul><a href="mailto:alice.cieslik@gmail.com?subject=Kontaktanfrage">E-Mail</a></ul>
+      <ul><a href="https://www.instagram.com/hello.itsalice/">Instagram: hello.itsalice</a></ul>
+      <ul>LinkedIn</ul>
+      <ul>Xing</ul>
+    </p>
+    </address>
```

Comment:
I think something went wrong here. 🤔 Your contact is currently not part of your footer, but should be nested inside your `footer` or inside your `main`. And you are using the multiple `ul` elements instead of `li` elements. The `p` element is also not necessary.